source/math/IntVectorList.ooc

```diff
@@ -0,0 +1,78 @@
+/*
+* Copyright (C) 2014 - Simon Mika <simon@mika.se>
+*
+* This sofware is free software; you can redistribute it and/or
+* modify it under the terms of the GNU Lesser General Public
+* License as published by the Free Software Foundation; either
+* version 2.1 of the License, or (at your option) any later version.
+*
+* This software is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+* Lesser General Public License for more details.
+*
+* You should have received a copy of the GNU Lesser General Public License
+* along with this software. If not, see <http://www.gnu.org/licenses/>.
+*/
+use ooc-collections
+import math
+import FloatVectorList
+
+IntVectorList: class extends VectorList<Int> {
+	init: func ~default {
+		this super()
+	}
+	init: func ~heap (capacity: Int) {
+		super(capacity)
+	}
+	init: func ~fromVectorList (other: VectorList<Int>) {
+		this super(other _vector)
```

Comment:
Oh, just one more thing before I'm happy. `super`, not `this super`. You probably copied this from `FloatVectorList` - I have a PR which fixes that over there.